src/Todoist.Net/Models/User.cs

```diff
@@ -1,28 +1,51 @@
+using System.Collections.Generic;
+using System.Reflection;
 using System.Text.Json.Serialization;
 
 namespace Todoist.Net.Models
 {
     /// <summary>
     /// Represents a Todoist user.
     /// </summary>
-    /// <seealso cref="Todoist.Net.Models.UserBase" />
-    public class User : UserBase
+    public class User : ICommandArgument, IUnsettableProperties
```

Comment:
Should we use BaseUnsetEntity here?

Reply:
Well, I found that the `UserBase` entity didn't inherit from the `BaseEntity` class.
It's mainly because the `Id` property isn't required, not until the `UserInfo` model.

Reply:
Got it